src/tools/rust-analyzer/crates/ide-diagnostics/src/handlers/unresolved_macro_call.rs

```diff
@@ -94,7 +94,6 @@ macro_rules! panic {
 }
 
 //- /lib.rs crate:foo deps:core
-#[macro_use]
```

Comment:
This needs to be undone, the test does not run against the standard library here